payments-core/src/main/java/com/stripe/android/payments/core/authentication/PaymentAuthenticator.kt

```diff
@@ -31,11 +31,9 @@ abstract class PaymentAuthenticator<Authenticatable> : ActivityResultLauncherHos
         requestOptions: ApiRequest.Options
     ) {
         val lifecycleOwner = host.lifecycleOwner
-        lifecycleOwner.lifecycleScope.launch {
```

Comment:
This behavior change might not be intentional.

Reply:
Gave this a quick test run and it seems to work fine when running in the existing scope.

Reply:
Gave this a quick test run and it seems to work fine when running in the existing scope.

Reply:
Same! I also ran through the code to make sure it was logical. But I think this was really the only "change", so figured I'd call it out.